scripts/squads.yml

```diff
@@ -0,0 +1,224 @@
+  - &Anil
+    name: Anil Puttabuddhi
+    github: anilputtabuddhi
+  - &Ben
+    name: Ben Henshall
+    github: Ben-Henshall
+    twitter: ben_henshall
+  - &Catarina
+    name: Ana Catarina Figueiredo
+    github: AnnKatF
+    twitter: AnnKatFig
+  - &Chesley
+    name: Chesley Stephens
+    github: cstephens-babylon
+  - &Chitra
+    name: Chitra Kotwani
+    github: chitrakotwani
+    twitter: chitrakotwani
+  - &DanielH
+    name: Daniel Haight
+    github: Daniel1of1
+  - &DanielS
+    name: Daniel Spindelbauer
+    github: sdaniel55
+    twitter: sdaniel55
+  - &Danilo
+    name: Danilo Aliberti
+    github: daniloaliberti
+  - &David
+    name: David Rodrigues
+    github: dmcrodrigues
+    twitter: dmcrodrigues
+  - &Diego
+    name: Diego Petrucci
+    github: diegopetrucci
+    twitter: diegopetrucci
+  - &Emese
+    name: Emese Toth
+    github: emeseuk
+  - &Giorgos
+    name: Giorgos Tsiapaliokas
+    github: gtsiap
+  - &Ilya
+    name: Ilya Puchka
+    github: ilyapuchka
+    twitter: ilyapuchka
+  - &Inami
+    name: Yasuhiro Inami
+    github: inamiy
+    twitter: inamiy
+  - &James
+    name: James Birtwell
+    github: jimmybee
+  - &Joao
+    name: João Pereira
+    github: NSMyself
+    twitter: NSMyself
+  - &Joshua
+    name: Joshua Simmons
+    github: j531
+  - &Julien
+    name: Julien Ducret
+    github: brocoo
+  - &Konrad
+    name: Konrad Muchowicz
+    github: konrad-em
+  - &Martin
+    name: Martin Nygren
+    github: zzcgumn
+  - &Michael
+    name: Michael Brown
+    github: mluisbrown
+    twitter: mluisbrown
+  - &Michal
+    name: Michał Kwiecień
+    github: MichalTKwiecien
+    twitter: kwiecien_co
+  - &Olivier
+    name:  Olivier Halligon
+    github: AliSoftware
+    twitter: aligatr
+  - &Patrick
+    name: Patrick Westmeyer
+    github: bh-pwestmeyer
+  - &Rui
+    name: Rui Peres
+    github: RuiAAPeres
+    twitter: peres
+  - &Sam
+    name: Sam Francis
+    github: SamFrancis-Babylon
+  - &Sergey
+    name: Sergey Shulga
+    github: sergdort
+    twitter: SergDort
+  - &Simon
+    name: Simon Cass
+    github: scass91
+    twitter: codercass
+  - &Viorel
+    name: Viorel Mihalache
+    github: viorel15
+    twitter: viorelMO
+  - &Witold
+    name: Witold Skibniewski
+    github: mr-v
+  - &Yuri
+    name: Yuri Karabatov
+    github: karabatov
+    twitter: karabatov
+
+
+################################################################
+# Global roles transverse to the squads
+#
+roles:
+  - name: Engineering Manager
+    members:
+      - *Rui
+  - name: Chapter Leads
+    members:
+      - *Catarina
+      - *David
+  - name: Squad Tech Leads
+    members:
+      - *Michael
+
+
+################################################################
+# List of the Squads and their members
+#
+squads:
+  - name: Appointments
+    desc: Appointment booking.
+    members:
+      - *Witold
+  - name: Consultation
+    desc: Flows for all the consultation like prescriptions and video consultation.
```

Comment:
There is a separate squad for prescriptions now. Even that is not under our purview anymore. Consultation experience takes care of audio/ video consultation, consultation feedback, pre-consultation work like getting the user ready for consultation, informing about lateness, educating the patient about what to expect and how to be prepared for consultation. Lot of it is planned work

Reply:
@chitrakotwani how is this split then? Adrian in Consultation and you in Prescriptions? You both in Prescriptions and nobody from iOS in Consultation anymore?
@AnnKatF can I let you update the yml accordingly?

Reply:
Both Adrian and I are in consultations. Adam and Konrad are in prescriptions. This is mentioned on this page BTW

Reply:
I will proceed with the changes.

Reply:
Oh ok so the 2 squads themselves and their respective members are correct, only the squad description for Consultations is to be updated, right :+1:

Reply:
Sorry for the confusion 😞 Let me just add suggestion instead